pkg/csi/service/cns/nodes.go

```diff
@@ -58,7 +58,20 @@ func (nodes *Nodes) nodeAdd(obj interface{}) {
 }
 
 func (nodes *Nodes) nodeUpdate(oldObj interface{}, newObj interface{}) {
-	nodes.nodeRegister(newObj)
+	newNode, ok := newObj.(*v1.Node)
+	if newNode == nil || !ok {
```

Comment:
The `newNode == nil` isn't needed as that is always the case if `!ok`.

Reply:
Done